superset/errors.py

```diff
@@ -164,14 +165,13 @@ class SupersetErrorType(str, Enum):
             ),
         },
     ],
-    SupersetErrorType.TEST_CONNECTION_INVALID_PASSWORD_ERROR: [
+    SupersetErrorType.TEST_CONNECTION_ACCESS_DENIED_ERROR: [
```

Comment:
Same thing, your PR is trying to override an existing error. You need to change this to 1014 and add it below.

Reply:
This still needs to be fixed.